samples/tab-device-permissions/nodejs/src/components/Tab.tsx

```diff
@@ -6,8 +6,11 @@ import { Grid, Segment } from '@fluentui/react-northstar'
 import ScanBarCode from './ScanBarCode';
 import GetGeoLocation from './GetGeoLocation';
 import CaptureImage from './CaptureImage';
+import CaptureImageDesktop from './CaptureImageDesktop';
 import PeoplePicker from './PeoplePicker';
 import CaptureAudio from './CaptureAudio';
+import CaptureAudioDesktop from './CaptureAudioDesktop';
+import CaptureVideoDesktop from './CaptureVideoDesktop';
 import GetLocationDesktop from './GetLocationDesktop';
```

Comment:
Could you please also include Notifications? 
navigator.permissions.query({ name: 'notifications' });